travel-planner/travel-planner-backend/routes/maps.js

```diff
@@ -5,9 +5,62 @@ const bodyParser = require("body-parser");
 router.use(bodyParser.json());
 const Parse = require("parse/node");
 const axios = require("axios");
+// const {Client} = require("@googlemaps/google-maps-services-js");
 
 // create delete pin endpoint?
 
+router.post("/getDirections", async (req, res) => {
+	const type = req.body.type;
+	const origin = req.body.origin;
+	const destination = req.body.destination;
+	const distance = req.body.distance;
+	const duration = req.body.duration;
+	const directionsArr = req.body.directions;
```

Comment:
I like your style here. But I wanted to let you know of a new syntax `const {type, origin} = req.body;` which extracts properties of 1 object in 1 line. I say I like  your way because it wouldn't fit 1 line anyway and is quite readable as is.